goal_orientation:
  w: 0.9776415253343371
  z: -0.2102785009028517
goal_position:
  x: 0.920342472268002
  y: 0.7403905844987336
